etl/src/main/scala/etl/ETL.scala

```diff
@@ -39,6 +39,32 @@ object Progress {
   }
 }
 
+object Images {
+
+  private def groupArticles(articlesWithoutImages: List[String]): List[String] = {
+    val groups = articlesWithoutImages.grouped(10).toList
+    groups.map(group => group.mkString(","))
+  }
+
+  private def getArticleResponse(articleIds: String)(implicit capiClient: GuardianContentClient): Future[SearchResponse] = {
+    capiClient.getResponse(capiClient.search.ids(articleIds).showElements("image"))
+  }
+
+  def getArticlesMissingImages()(implicit db: DB, capiClient: GuardianContentClient): Future[List[SearchResponse]] = {
+    val articlesMissingImages = (db.getArticlesWithRecipes().toSet diff db.getArticlesWithSavedImages().toSet).toList
+    val articleIdsParamGroups: List[String] = groupArticles(articlesMissingImages)
+
+    Future.sequence(articleIdsParamGroups.map(group => getArticleResponse(group)))
+  }
+
+  def processArticles(articles: List[SearchResponse])(implicit db: DB) {
```

Comment:
It only inserts images if it can find them for the given article. Hopefully most of the time when this runs it will be because the article doesn't have images... so I've gone for `insertImagesFromArticles`